packages/adminrouter/windows.build.ps1

```diff
@@ -1,3 +1,19 @@
-$ErrorActionPreference = "stop"
-New-Item -ItemType Directory "$env:PKG_PATH/bin/Apache24/conf"
-Copy-Item "pkg/extra/apache-windows/adminrouter.conf" "$env:PKG_PATH/bin/Apache24/conf"
+$ErrorActionPreference = "stop"
+New-Item -ItemType Directory -Path "$env:PKG_PATH/etc/", "$env:PKG_PATH/etc/dcos-adminrouter/"
+New-Item -ItemType Directory -Path "$env:PKG_PATH/etc/dcos-adminrouter/conf/", "$env:PKG_PATH/etc/dcos-adminrouter/logs/"
+New-Item -ItemType Directory -Path "$env:PKG_PATH/bin/", "$env:PKG_PATH/bin/dcos-adminrouter/"
+New-Item -ItemType Directory "C:\Temp"
```

Comment:
This now fails, at it already exists. Maybe add `-ErrorAction Ignore` to the command, in case it does not exist.

Reply:
Added checking for an existing folder